vips/vip-converter/bscmainnet.ts

```diff
@@ -0,0 +1,68 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const SINGLE_TOKEN_CONVERTER_BEACON = "0x4c9D57b05B245c40235D720A5f3A592f3DfF11ca";
+export const NEW_SINGLE_TOKEN_CONVERTER_IMP = "0x0d8C55db1C0778A094B0283ed9e6bA2680B5C823";
+export const NEW_RISK_FUND_CONVERTER_IMP = "0xda35a8DB2031e6d8164Eb8aCCd2C2f83c52E7745";
+export const RISK_FUND_CONVERTER_PROXY = "0xA5622D276CcbB8d9BBE3D1ffd1BB11a0032E53F0";
+export const PROXY_ADMIN = "0x6beb6D2695B67FEb73ad4f172E8E2975497187e4";
+
+export const PROXY_ADMIN_LIQUIDATOR = "0x2b40B43AC5F7949905b0d2Ed9D6154a8ce06084a";
+export const LIQUIDATOR_CONTRACT = "0x0870793286aaDA55D39CE7f82fb2766e8004cF43";
+
+export const OLD_IMPL = "0xE26cE9b5FDd602225cCcC4cef7FAE596Dcf2A965";
+export const TEMP_IMPL = "0x3aD4b5677AdC2a6930B2A08f443b9B3c6c605CD8";
+export const PSR = "0xCa01D5A9A248a830E9D93231e791B1afFed7c446";
```

Comment:
I think we should set and use the common addresses from the networkAddresses helper.

Reply:
Done https://github.com/VenusProtocol/vips/pull/201/commits/1e85ff6f2742126fe8f5d365eb5fe28308c073e5

Reply:
I think the agreement was to add and use addresses from the networkAddresses file only if those addresses are needed/used by the VIP framework. If one VIP needs an address, we should specify that address in the VIP file or in the simulation file

In this case, AFAIK, the VIP framework doesn't use the addresses for ProtocolShareReserve, RiskFundConverter or SingleTokenConverterBeacon. So, I would remove them from the networkAddresses file